getLanguageServer.sh

```diff
@@ -35,17 +35,33 @@ VERSION=$(curl -sSL --compressed https://static.snyk.io/snyk-ls/$PROTOCOL_VERSIO
 DESTINATION="/usr/local/bin/snyk-ls"
 DOWNLOAD_URL="https://static.snyk.io/snyk-ls/$PROTOCOL_VERSION/snyk-ls_${VERSION}_${OS}_${ARCH}"
 
+set +e
+if [[ -f $DESTINATION ]]; then
```

Comment:
Question: if the check below realizes that `DESTINATION` is not accessible, this logic is not executed for `$PWD/snyk-ls`. does it make sense to move the condition in line 48 up?